### DISCUSSION TITLE
自定义构建镜像在主页消失

### DISCUSSION THREAD

**Discussion by 498777** [immortalwrt/immortalwrt#1413] [Q&A]
### Describe the bug

![bug](https://github.com/user-attachments/assets/19d3f445-e775-4c55-879a-060d1d794ca4)

### ImmortalWrt version

ImmortalWrt SNAPSHOT r31191-300a3d4f2d / LuCI Master 24.209.33969~31219a4

### ImmortalWrt release

ImmortalWrt SNAPSHOT r31191-300a3d4f2d / LuCI Master 24.209.33969~31219a4

### ImmortalWrt target/subtarget

ImmortalWrt SNAPSHOT r31191-300a3d4f2d / LuCI Master 24.209.33969~31219a4

### Device

x86/64

### Image kind

Official downloaded image

### Steps to reproduce

自定义构建镜像在主页消失，无法自定义构建镜像，服务器地址不可达，此功能下架还是维护？时长大约是？

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by shaw21003** [comment]
前两天我也没找到，自己配了一天默认的，后来自定义构建又出现了！！被坑了！
